sender-app/configuration/application.properties

```diff
@@ -129,6 +129,11 @@ debezium.db.password=
 # debezium MySQL connector property named snapshot.mode
 debezium.snapshotMode=schema_only
 
+#Controls whether and how long the connector holds the global MySQL read lock, which prevents any updates to
```

Comment:
We need to update the code to read in this property

Reply:
I believed it was automatically loaded via Spring Boot magic... Should have a better look

Reply:
how is it done for  debezium.snapshotMode for instance  so that I can have a look ?

Reply:
This is not a spring boot property, it actually a custom property, extended is just the default value, it is not like it is getting read in. It is set here https://github.com/openmrs/openmrs-eip/blob/master/openmrs-watcher/src/main/java/org/openmrs/eip/mysql/watcher/DebeziumRoute.java#L31

Reply:
Yes I know it: but you mentionned that we need to update the code to read it and I don't understand this point as I did it in a PR for openmrs-eip: https://github.com/openmrs/openmrs-eip/pull/28/files#